### PR TITLE
use native dem (30m) for the calculation, resample the product with p…

### PR DESCRIPTION
**Pull request by cirrusasf**
…ixel_size determined by looks

### CODE REVIEW

File: src/hyp3_isce2/insar_tops_burst.py [hunk 1]
```diff
@@ -242,6 +244,7 @@ def make_parameter_file(
         'Azimuth bandpass filter: no\n',
         f'DEM source: {dem_name}\n',
         f'DEM resolution (m): {dem_resolution}\n',
+        f'Pixel Size (m): {pixel_size}\n',
```
Comment (asjohnston-asf) [inline src/hyp3_isce2/insar_tops_burst.py:247]: I vote against adding the pixel size to the parameter .txt file unless someone has a compelling use case for it. Pixel size is a self-describing property of the output geotiffs, so I don't think the extra code to include it in the txt file is justified. Thoughts?

File: src/hyp3_isce2/insar_tops_burst.py [hunk 2]
```diff
@@ -371,6 +382,8 @@ def main():
 
     args = parser.parse_args()
 
+    pixel_size = get_pixel_size(args.looks)
```
Comment (asjohnston-asf) [inline src/hyp3_isce2/insar_tops_burst.py:385]: Good practice is to wait to define variables until they're needed to reduce the number of lines a reader needs to keep in their head at once. This is better moved to just before the `translate_outputs()` call on line 414.

File: src/hyp3_isce2/insar_tops_burst.py (hunk 1 above)
Comment (cirrusasf) [reply src/hyp3_isce2/insar_tops_burst.py]: will delete the pixel size in the txt file

File: src/hyp3_isce2/insar_tops_burst.py [hunk 3]
```diff
@@ -414,7 +424,8 @@ def main():
         secondary_scene=secondary_scene,
         azimuth_looks=azimuth_looks,
         range_looks=range_looks,
-        swath_number=swath_number
+        swath_number=swath_number,
+        pixel_size=pixel_size
```
Comment (asjohnston-asf) [inline src/hyp3_isce2/insar_tops_burst.py:428]:
```suggestion
        swath_number=swath_number
```

File: src/hyp3_isce2/insar_tops_burst.py [hunk 4]
```diff
@@ -167,6 +167,7 @@ def make_parameter_file(
         range_looks: Number of range looks
         dem_name: Name of the DEM that is use
         dem_resolution: Resolution of the DEM
+        pixel_size: pixel spacing of output product
```
Comment (asjohnston-asf) [inline src/hyp3_isce2/insar_tops_burst.py:170]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.6.0]
+### Changed
+* Pixel size of output products is set to be the same resolution as Geocoding DEM
```
Comment (asjohnston-asf) [inline CHANGELOG.md:11]:
```suggestion
* Pixel sizes of output GeoTIFFs now better reflect their resolution: 80m for 20x4 looks, 40m for 10x2 looks, and 20m for 5x1 looks.
```

**Comment by asjohnston-asf** [comment]
With the changelog tweak I'll be content to move this into develop and release shortly after. I'll want to document that we're likely losing some resolution for 5x1 products by geocoding at ~27m before resampling to 20m, but I'm content to revisit that issue later.

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**